tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
That doesn't sound right to me - nothing should be using it as a reference. Can you show what the error was?